doc/classes/OS.xml

```diff
@@ -39,6 +40,7 @@
 			<description>
 				Creates a new instance of Godot that runs independently. The [param arguments] are used in the given order and separated by a space.
 				If the process creation succeeds, the method will return the new process ID, which you can use to monitor the process (and potentially terminate it with [method kill]). If the process creation fails, the method will return [code]-1[/code].
+				See [method create_process] if you wish to run a different process.
 				[b]Note:[/b] This method is implemented on Android, iOS, Linux, macOS and Windows.
```

Comment:
```suggestion
				[b]Note:[/b] This method is implemented on Android, Linux, macOS and Windows.
```
None of the process creation methods are available on non-jailbroken iOS.

Reply:
Oops. This removal was likely accidental.

Reply:
Huh. Heh. I didn't touch that one. Glad these PRs force people to pay attention